action.yml

```diff
@@ -189,6 +189,10 @@ runs:
         echo "${GEMINI_ERRORS}" >> "${GITHUB_OUTPUT}"
         echo "EOF" >> "${GITHUB_OUTPUT}"
 
+        if echo "${GEMINI_ERRORS}" | grep -q "Reached max session turns for this session"; then
```

Comment:
if the error message`"Reached max session turns for this session"` changes, then this stops working, right?